pkg/simple/client/monitoring/metricsserver/metricsserver.go

```diff
@@ -194,24 +193,29 @@ func (m metricsServer) getPodMetricsFromMetricsAPI(edgePods map[string]bool, pod
 			}
 			combinedPodMetrics = append(combinedPodMetrics, *metrics)
 		}
-		return combinedPodMetrics, isPodMetricsUsingListRequst, nil
+		return combinedPodMetrics, nil
 	}
 
 	// use list request in anthor cases
```

Comment:
use list request in anthor cases
=>
use list request for other cases

Reply:
fixed